app/preload.js

```diff
@@ -33,6 +35,15 @@ function getArtworkURL() {
   return null
 }
 
+function getLikeStatus() {
+  const liked = document.querySelector('.sc-button-like.playbackSoundBadge__like.sc-button-selected')
```

Comment:
I think this selector would be less fragile this way: `.playbackSoundBadge__actions .playbackSoundBadge__like.sc-button-selected`. That would also make it more similar to the one used in `getArtworkURL` above.